components/clp-mcp-server/tests/server/test-utils.py

```
@@ -0,0 +1,130 @@
+
+    def test_invalid_timestamp_type(self):
+        """Validates the handling of non integer timestamp types."""
+        result = filter_query_results(self.INVALID_TYPE_ENTRIES)
+
+        assert result == self.INVALID_TYPE_EXPECTED
+
+    def test_invalid_timestamp_value(self):
+        """Validates the handling of invalid timestamp values."""
+        result = filter_query_results(self.INVALID_VALUE_ENTRIES)
+
+        assert result == self.INVALID_VALUE_EXPECTED
```

Ruff (0.14.1)

No newline at end of file

Add trailing newline

(W292)